db/db_test_util.cc

```diff
@@ -1169,7 +1169,7 @@ void DBTestBase::validateNumberOfEntries(int numValues, int cf) {
   int seq = numValues;
   while (iter->Valid()) {
     ParsedInternalKey ikey;
```

Comment:
initialize ikey with sequence = 0 with constructor, then remove the next line.

Reply:
Would .clear() be fine?

Reply:
Maybe just remove the next line. Let see if clang_analyzer complains..